src/ITensorNetworks.jl

```diff
@@ -4,17 +4,20 @@ using DataGraphs
 using Dictionaries
 using DocStringExtensions
 using Graphs
+using IsApprox
 using ITensors
 using ITensors.ContractionSequenceOptimization
 using ITensors.ITensorVisualizationCore
 using MultiDimDictionaries
 using NamedGraphs
 using Requires
+using SparseArrayKit
```

Comment:
Is this used anywhere?

Reply:
Only in an intermediate internal step, namely the conversion of an `OpSum` to a `DataGraph` of `SparseArray`s with operators as entries used to represent a sparse symbolic tree tensor network Hamiltonian. This format was just the easiest for visualization and debugging for me personally, but I'm definitely open to suggestions for a different format (if we even want to keep this intermediate representation around at all in the end).

Reply:
Ok, just wanted to double check, that sounds reasonable for now.